pact.cabal

```diff
@@ -64,6 +64,22 @@ library pact-prettyprinter
     , ansi-terminal >=0.4
     , prettyprinter >= 1.7
 
+-- -------------------------------------------------------------------------- --
+-- Internal: unsafe functions from base
+--
+-- This is to avoid cluttering production code with
+-- `{-# OPTIONS_GHC -Wno-x-partial #-}` pragmas with base >= 4.20
```

Comment:
Why not disable it package-wide?

Reply:
I think that it's generally good to keep the scope of disabled warnings narrow. If it was possible to scope it to an individual functions, I would have preferred that -- in that case this library would probably not be necessary, because one could annotate each use case in the original code.